src/containers/navigation/navContainer.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import NavBar from '../../components/navigation/navBar';
+
+class Navigation extends React.Component {
+	render() {
+		return (
+			<div>
```

Comment:
prefer React.Fragment instead of the extra div

Reply:
Since I have only one element, I will leave it without the <div> as react.frangment is aimed at grouping multiple elements